tests/test_reporting_config_exports.py

```
@@ -80,6 +80,10 @@ def import_branding_config(self, path, client_name):
 
 
 class TestExportChartTypeConfig:
+    @pytest.fixture(autouse=True)
+    def _require_matplotlib(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_require_matplotlib`

Add return type annotation: `None`

(ANN202)